Makefile

```diff
@@ -69,25 +73,30 @@ install:
 
 lint:
 	go get -v github.com/golang/lint/golint
-	for file in $$(find . -name '*.go' | grep -v vendor | grep -v '\.pb\.go' | grep -v '\.pb\.gw\.go'); do \
+	for file in $$(find . -name '*.go' | grep -v vendor | \
+		                                   grep -v '\.pb\.go' | \
```

Comment:
This won't build

Reply:
It does. Just tested again using `make lint`. It shows up like this here because of the tab characters.

Reply:
Oh the "do" didn't show in the review because its way on the right. The formatting is still messed up though. Please fix this.